azure/route.sh

```diff
@@ -12,6 +12,34 @@ fn azure_route_table_delete(name, group) {
 	)
 }
 
+fn azure_route_table_route_new(name, group, routetable, address, hoptype) {
+	instance = (
+		"--name"
+		$name
+		"--resource-group"
+		$group
+		"--route-table-name"
+		$routetable
+	    "--address-prefix"
+		$address
+	    "--next-hop-type"
+		$hoptype
+	)
+
+	return $instance
+}
+
+fn azure_route_table_route_set_hop_address(instance, hopaddress) {
```

Comment:
I'm still trying to understand the whole thing, but it seems like I could add multiple hop addresses. If that is the case, it seems more clear to call:

```
azure_route_table_route_add_hop_address
```

Instead of set, calling set conveys the idea that calling it will overwrite a previous value.

If it is always just one hop-address, it would make more sense to pass it on new, setting stuff around does not seem very useful on this case.